python/cudf/cudf/utils/dtypes.py

```diff
@@ -154,6 +154,20 @@ def is_numerical_dtype(obj):
     return dtype.kind in "biuf"
 
 
+def is_integer_dtype(obj):
+    try:
+        dtype = np.dtype(obj)
+    except TypeError:
+        return pd.api.types.is_integer_dtype(obj)
+    return dtype.kind in "iu"
```

Comment:
Is there a reason to try `np.dtype(obj)` here now instead of just using `pd.api.types.is_integer_dtype`?

Reply:
Yes, we're trying to avoid the overhead of the Pandas util when possible. This should be faster in many common cases.